workflows/data_io_utils/schemas/assembly.py

```diff
@@ -650,3 +651,76 @@ def __init__(self):
                 annotation_summary_dir,
             ],
         )
+
+
+class StudySummary(BaseModel):
+    """Schema for study summary files."""
+
+    source: str = Field(..., description="Analysis source identifier")
+    short_description: str
+    long_description: str
+
+    def matches_file(self, file_path: Path) -> bool:
+        """
+        Check if a file path matches this summary type.
+
+        Expected pattern: {study_accession}_{source}_study_summary.tsv
+        """
+        return f"_{self.source}_study_summary.tsv" in file_path.name
```

Comment:
maybes `endswith` just for even more explicitness. protects against accidental change in gzipiness.